extensions/common/sql/sql-core/src/main/java/org/eclipse/edc/sql/statement/SqlExecuteStatement.java

```diff
@@ -0,0 +1,65 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.sql.statement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.String.format;
+import static java.util.stream.Collectors.joining;
+import static org.eclipse.edc.sql.statement.ColumnEntry.standardColumn;
+
+public class SqlExecuteStatement {
```

Comment:
doc? Maybe `SqlModificationStatement` would be a more suitable name, to mirror the `SqlQueryStatement`?

Reply:
I took the name from the methods in `SqlQueryExecutor`: `execute` (to modify) and `query` (to get), I think it make sense to stay consistent, I'm open to modifications but on both sides eventually